src/playground/service.clj

```diff
@@ -63,8 +69,8 @@
   (session-backend
    {:authfn (fn [request]
               (let [{:keys [username password]} request]
```

Comment:
`request` doesn't have username and password fields (or it shouldn't have).

Very likely this fn is dealing with the value of `(get-in request [:session :identity])`.

Please verify it (scientifically) and then rename `request` here

Reply:
This backend and the authentication-interceptor is not being used anywhere. The session interceptor creates the session and the login-endpoint populates the identity map that is later checked.

Reply:
Managed to figure out that all the backend does is returning true/nil . What I don't understand is why the interceptor isn't enough to check if `:identity` is  true same the `authorization-interceptor` does. What is the point of having the backend in a separate def?

Reply:
Yes, I would try deleting `session-auth-backend` and the call `buddy.auth/authenticated?`.

Given you set yourself the :identity, you can also read it yourself. So no need for Buddy here apparently

Reply:
perfect!